README.md

```diff
@@ -1,2 +1,22 @@
-# S-BLH-roman-numeral-converter
-2nd project of java script
+# PRoman-Numeral-Converter 🚀
+
+## Project Description 📝
+
+
+- 2nd project of js
+- its a converter , that can change latin numbers to roman numbers
+
+## Technologies Used 🛠️
+
+
+- HTML
+- CSS
+- JavaScript
+
+## Author 👩‍💻
+
+- LinkedIn: https://www.linkedin.com/in/sumita-balouch-959197309?utm_source=share&utm_campaign=share_via&utm_content=profile&utm_medium=ios_app
+- Email: sumitabalouch@gmail.com
+
+## License 📜
+MIT license.
```

Comment:
- Add a demo section with a screenshot of your project and also add How to use section to your README.md file